apis/metal3.io/v1alpha1/baremetalhost_types.go

```diff
@@ -722,12 +735,16 @@ type BareMetalHostStatus struct {
 	// Important: Run "make generate manifests" to regenerate code
 	// after modifying this file
 
-	// OperationalStatus holds the status of the host
+	// OperationalStatus holds the status of the host and its
+	// validated by the checkStatusAnnotation function in
+	// baremetalhost_validation.go
```

Comment:
This will be a part of auto-generated docs. I don't think we should refer to files there.

Reply:
Removed the file reference.